src/main/groovy/liquibase/harness/data/ChangeDataTests.groovy

```diff
@@ -102,11 +110,33 @@ class ChangeDataTests extends Specification {
             //For embedded databases, let's create separate connection to run checking SQL
             if (shouldOpenNewConnection(connection, "sqlite", "snowflake", "postgres", "oracle", "mysql")) {
                 newConnection = DriverManager.getConnection(testInput.url, testInput.username, testInput.password)
+                
```

Comment:
what do those pieces do for the test? We already have this statement in 
https://github.com/liquibase/liquibase/blob/37c469eef25870102b11da27b6b1c8e5e314bda6/liquibase-snowflake/src/main/java/liquibase/database/core/SnowflakeDatabase.java#L185-L196